test_communication/CMakeLists.txt

```diff
@@ -136,7 +136,7 @@ if(BUILD_TESTING)
     endforeach()
 
     # TODO different vendors can't talk to each other right now
-    if("${rmw_implementation1} " STREQUAL "${rmw_implementation2} " OR ("${rmw_implementation1} " STREQUAL "rmw_connext_cpp " AND "${rmw_implementation2} " STREQUAL "rmw_connext_dynamic_cpp ") OR ("${rmw_implementation1} " STREQUAL "rmw_connext_dynamic_cpp " AND "${rmw_implementation2} " STREQUAL "rmw_connext_cpp "))
+    if(rmw_implementation1 STREQUAL rmw_implementation2 OR (rmw_implementation1 STREQUAL "rmw_connext_cpp" AND rmw_implementation2 STREQUAL "rmw_connext_dynamic_cpp") OR (rmw_implementation1 STREQUAL "rmw_connext_dynamic_cpp" AND rmw_implementation2 STREQUAL "rmw_connext_cpp"))
```

Comment:
That's fine, I was just wondering if it was long before because we weren't supposed to do multiline if statements before (if older versions of CMake didn't fully support it or something).